bioblend/_tests/TestGalaxyQuotas.py

```diff
@@ -0,0 +1,59 @@
+"""
+Use ``nose`` to run these unit tests.
+"""
+import uuid
+
+from . import GalaxyTestBase, test_util
+
+
+@test_util.skip_unless_galaxy()
+class TestGalaxyQuotas(GalaxyTestBase.GalaxyTestBase):
+
+    def setUp(self):
+        super(TestGalaxyQuotas, self).setUp()
+        # Quota names must be unique, and they're impossible to delete
+        # without accessing the database.
+        self.quota_name = 'BioBlend-Test-Quota-%s' % uuid.uuid4().hex
+        self.quota = self.gi.quotas.create_quota(
+            self.quota_name, 'testing', '100 GB', '=',
+            default='registered'
+        )
+
+    def tearDown(self):
+        self.gi.quotas.update_quota(self.quota['id'], default='registered')
```

Comment:
yes, unfortunately so. If `default!="no"`, then you can make any number of update_quota calls that you want. However, if `default=no` and you make a second update_quota call with default=no, then it will return an HTML error page. Documented in the [docstring](https://github.com/galaxyproject/bioblend/pull/226/files#diff-df5232b41baccd7cd9c8aa59f76b497fR139)